packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -188,6 +189,7 @@ BpkGraphicPromo.propTypes = {
   invertVertically: PropTypes.bool,
   textAlign: PropTypes.oneOf(Object.values(TEXT_ALIGN)).isRequired,
   textColor: PropTypes.oneOf(Object.values(TEXT_COLORS)),
+  style: PropTypes.string,
```

Comment:
I think this type should be `style: { [key: PropTypes.string]: PropTypes.string };`

Reply:
```typescript
style: PropTypes.shape({
  <something>
})
```